<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuraciones</class>
 <widget class="QDialog" name="Configuraciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuraciones</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Test Process</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Customer Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Path Salida</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Tester Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Board Style</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Assembly Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Nombre Equipo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Site</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="Line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="Site">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CustomName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CISCO SMT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CISCO</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Assembly_Name">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Operador ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Path Entrada</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Tester_Name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ICT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AOI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XRAY</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Board_Style">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TestProcess">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>analog</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Digital</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mixed</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Equipo_Name">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Assembly Rev</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 167);</string>
   </property>
   <property name="text">
    <string>Test Status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TestStatus">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fail</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Abort</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pass</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Short</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RevA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RevB</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Save_Button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="DirectIN"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="DirectSave"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_2">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Configuraciones</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Configuraciones</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
